include/bucket.h

```diff
@@ -61,12 +61,8 @@ namespace Bucket_Boruvka {
   inline static void update(vec_t& a, vec_hash_t& c, const vec_t& update_idx, const vec_hash_t& update_hash);
 } // namespace Bucket_Boruvka
 
-inline col_hash_t Bucket_Boruvka::col_index_hash(const unsigned bucket_col, const vec_t& update_idx, const long sketch_seed) {
-  struct {
-    unsigned bucket_col;
-    vec_t update_idx;
-  } __attribute__((packed)) buf = {bucket_col, update_idx};
-  return col_hash(&buf, sizeof(buf), sketch_seed);
+inline col_hash_t Bucket_Boruvka::col_index_hash(const vec_t& update_idx, const long seed_and_col) {
+  return col_hash(&update_idx, sizeof(update_idx), seed_and_col);
```

Comment:
is this the main change?

Reply:
This and the node_id_t -> size_t

Reply:
do u mind summarizing what this change does?

Reply:
So basically, instead of seeding by appending the sketch column to the beginning of the data we modify the sketch seed itself.

This reduces the size of the data and saves us the time to construct the packed data.